src/main/java/ben/tasks/Task.java

```diff
@@ -0,0 +1,72 @@
+package ben.tasks;
+
+/**
+ * Represents a generic task in the Ben task management application.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I like that your boolean variables are named to sound like booleans, as per the coding standards